pkg/cri/server/container_start.go

```diff
@@ -64,6 +64,9 @@ func (c *criService) StartContainer(ctx context.Context, r *runtime.StartContain
 			// Set container to exited if fail to start.
 			if err := cntr.Status.UpdateSync(func(status containerstore.Status) (containerstore.Status, error) {
 				status.Pid = 0
+				if status.StartedAt == int64(0) {
+					status.StartedAt = time.Now().UnixNano()
```

Comment:
The container doesn't seem to be started so is it right to fill StartedAt with time.Now?

Reply:
Maybe it's align with finishedAt. However, it's weird to set startAt since it doesn't start. For instance, container process is `/bin/404-not-found-command`. It won't start because no such command.

Reply:
Although it's weird to set startAt since it doesn't start, the reason here is to clearly define the starting point of container initiation. Following the same logic, if the container hasn't started, should there also be no finishedAt?